salt/haproxy/haproxy.yaml.jinja

```diff
@@ -55,10 +61,17 @@ spec:
         path: {{ pillar['ssl']['ca_file'] }}
 {% endif %}
         type: FileOrCreate
+    - name: ca-bundle-dir
+      hostPath:
+        path: {{ salt['file.dirname']( pillar['ssl']['sys_ca_bundle'] ) }}
```

Comment:
For consistency, shouldn't type: Directory be here after path?

Reply:
The default behavior is directory if the "thing" doesn't exist, and they same type of it does already exist. I think the typical expectation is that Directory be left out, but I do like being explicit. If it's changed here, though, it'll also need changed for the other directories in the other static pod definitions, both here and in container manifests (which are already merged and kicked upstream).

I don't really like "ugh, that's a pain" as reasoning, but I think "three PRs and some history mangling" means that might be where I land here. I'm now disappointed in myself. Thanks a lot. :D